jobs/config.json

```diff
@@ -2763,6 +2763,12 @@
     ], 
     "scenario": "kubernetes_e2e"
   }, 
+  "ci-perf-tests-kubemark-100-benchmark": {
+    "args": [
+      "./benchmark/runner.sh"
```

Comment:
It doesn't matter why it is used, just the fact that it needs to run docker, which containers cannot do